libs/cardano-ledger-core/src/Cardano/Ledger/Coin.hs

```diff
@@ -65,7 +67,7 @@ instance FromCBOR Coin where
   fromCBOR =
     ifDecoderVersionAtLeast
       (natVersion @9)
-      (Coin . fromIntegral <$> decodeWord64)
+      decodePositiveCoin
```

Comment:
This is wrong, please revert. Coins can actually be zero, because they are used in other places, not only in outputs 
```suggestion
      (Coin . fromIntegral <$> decodeWord64)
```

Reply:
Got you! Thanks!